Source/Constraint.swift

```diff
@@ -424,6 +425,7 @@ private extension NSLayoutAttribute {
                 case .Leading: return  (Config.interfaceLayoutDirection == .LeftToRight) ? insets.left : -insets.right
                 case .Trailing: return  (Config.interfaceLayoutDirection == .LeftToRight) ? insets.right : -insets.left
                 case .Width, .Height, .NotAnAttribute: return CGFloat(0)
+                default: return CGFloat(0)
```

Comment:
Are you sure these are required? Seems like there may be new attributes we need to manage? We don't want to miss any new ones.